readme.md

```diff
@@ -22,8 +22,8 @@ Hold `Alt` key (`Option` on MacOS) to measure distances in Figma.
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://entodasha.github.io/layout_stars/)
+- [TEST REPORT LINK](https://entodasha.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="937" alt="image" src="https://github.com/user-attachments/assets/14dfd698-7b98-476d-a189-a67e40a7f68d">